go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/fatih/color v1.14.1
 	github.com/fatih/structs v1.1.0
 	github.com/go-ldap/ldap/v3 v3.4.4
-	github.com/lestrrat-go/jwx v1.2.25
+	github.com/lestrrat-go/jwx v1.2.26
```

Comment:
@donatello send a separate PR for dependency update?

Reply:
ok sure

Reply:
This is done now, but CI will fail without the security fix.